### ISSUE TITLE
Add Conroy et al. (2010) attenuation model

### ISSUE THREAD

**Issue by karllark** [karllark/dust_attenuation#28]
There's a generalization of the CCM89 dust extinction model in the Appendix of Conroy+10 that varies the bump strength to allow for modeling attenuation in galaxies.  A possible addition for this package.  Originally proposed for the dust_extinction package, but more appropriate here.

https://github.com/karllark/dust_extinction/issues/53

**Comment by dcorre** [comment]
I don't think it should be in the dust_attenuation package but rather in the dust_extinction package.
Yes, it is used for galaxies but it is based on an extinction law (ccm89)...   If people want to use extinction recipes for galaxies, this is their choice they need to justify in their papers. We should clearly make a distinction between extinction and attenuation processes to avoid this usage. If I'm not mistaken that is one of the reason why you decided to create 2 separate packages.
However, I don't know whether this modified ccm89 brings something more than the more flexible FM recipes where you could only let the bump parameters vary and set all other parameters to the averaged MW values?

**Comment by karllark** [comment]
You make good points.  I have not looked at this work in detail yet.  As it was proposed for attenuation in galaxies, I figured it was best for this package.  But we should check.

I also have your concern that the FM90 and P92 already provide ways to vary the bump strength.  As does G16.  So, homework is needed clearly.

**Comment by dcorre** [comment]
From their work:
"We also consider attenuation curves whose functional form is equivalent
to the average extinction curve in the MW (Cardelli et al.
1989). As mentioned in Section 1, attenuation and extinction
are different concepts, but in simple geometries they can be
roughly equivalent (modulo wavelength dependence of the dust
albedo)."

I believe it strengthens the motivations which lead to the creation of this package ;)

Charlie Conroy used the Noll+09 formalism in a [more recent paper 2013 ](http://iopscience.iop.org/article/10.1088/2041-8205/775/1/L16/pdf) with Mariska Kriek. So I don't know whether they are still using the modified ccm89 for galaxies.

**Comment by karllark** [comment]
Hum... The clearly argues for adding the Noll model before this one.  I can contact Charlie and ask him what he thinks.

**Comment by karllark** [comment]
Really sounds like Noll is the way to go.
